spec/validation.spec.js

```diff
@@ -216,7 +216,7 @@ describe("Request validation", () => {
 
   it("should validate urls with special characters", () => {
     const specialRequestUrl = requestUrl + "&Body=It's+amazing";
-    const signature = "dsq4Ehbj6cs+KdTkpF5sSSplOWw=";
```

Comment:
Why there is a change in signature here?

Reply:
We had this discussion [here](https://github.com/twilio/twilio-node/pull/998#discussion_r1553172975)